server/s3_put_object_action_base.h

```diff
@@ -105,7 +105,7 @@ class S3PutObjectActionBase : public S3ObjectAction {
       std::shared_ptr<S3ObjectMetadata> &, struct s3_part_frag_context);
   void add_parts_fragment_oid_to_probable_dead_oid_list_success();
 
-  void startcleanup() final;
```

Comment:
Any reason understood yet on why it was kept final, which cant be overridden.

Reply:
Not sure, @knrajnambiar76 @DPG17 is there any reason?